providers/src/airflow/providers/edge/worker_api/routes/_v2_routes.py

```diff
@@ -90,6 +98,44 @@ def set_state_v2(worker_name: str, body: dict[str, Any], session=NEW_SESSION) ->
         return e.to_response()  # type: ignore[attr-defined]
 
 
+@provide_session
+def job_fetch_v2(worker_name: str, body: dict[str, Any] | None = None, session=NEW_SESSION) -> Any:
```

Comment:
Isn't the return type deterministic: `EdgeJobFetched | None`?

Reply:
That would be the type in FastAPI. But here we are in the "compatability leyer" which uses the fucntions of FastAPI and wraps into the same endpoint for Connexion. Connexion gets the JSON model as dump, not a pydantic object.